CHANGELOG.md

```diff
@@ -24,8 +24,11 @@ docs/releases/Hotfix-Documentation-Best-Practices.md
 
 INTERNAL NOTE
 -->
+## Flutter 3.29 Changes
+
+### [3.29.2](https://github.com/flutter/flutter/releases/tag/3.29.2)
 
-## Flutter 3.29.1 Changes
+- [flutter/163175](https://github.com/flutter/flutter/issues/163175) Improve performance of CanvasKit rendering.
```

Comment:
Can you update this to meet our changelog guidance? 
https://github.com/flutter/flutter/blob/master/docs/releases/Hotfix-Documentation-Best-Practices.md

My suggestion is based on reading the issue/pr and may not be correct. 
```suggestion
- [flutter/163175](https://github.com/flutter/flutter/pull/163175) Improve performance of CanvasKit rendering for web.
```